src/langchain_google_bigtable/document_loader.py

```diff
@@ -132,6 +146,20 @@ def lazy_load(
                         ].value,
                         mapping,
                     )
+            if (
+                self.metadata_as_json_column_family is not None
+                and self.metadata_as_json_column_name is not None
+                and self.metadata_as_json_column_family in row.cells
+                and self.metadata_as_json_column_name.encode()
+                in row.cells[self.metadata_as_json_column_family]
+            ):
+                metadata.update(
+                    json.loads(
+                        row.cells[self.metadata_as_json_column_family][
+                            self.metadata_as_json_column_name.encode()
+                        ][0].value.decode(self.metadata_as_json_encoding.value)
+                    )
+                )
```

Comment:
I'm inclined to think we should load this as metadata first, and overwrite the columns into it

Reply:
Done